### PR TITLE
Atlassian c2c Source

### PR DESCRIPTION
**Pull request by JV0812**
## Purpose of this pull request

This pull request is to create source doc for our new Atlassian C2C source.

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions, updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

## Ticket (if applicable)

[C2C - Atlassian - Update Docs](https://sumologic.atlassian.net/browse/CONN-3351)

### CODE REVIEW

**Comment by JV0812** [comment]
Converting this PR to draft as we are still unsure about the release date of this source.

**Comment by jpipkin1** [comment]
@JV0812, I added this new source to the product list (see commit "[Add Atlassian Source to product list](https://github.com/SumoLogic/sumologic-documentation/pull/4313/commits/02d862feb5f1c1fe69a605e6e0b72a536931531c)").

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/atlassian-source.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+<head>
+  <meta name="robots" content="noindex" />
+</head>
+
+<p><a href="/docs/beta"><span className="beta">Beta</span></a></p>
+
+<img src={useBaseUrl('img/send-data/atlassian-icon.png')} alt="atlassian-icon" width="40" />
+
+The Atlassian Organizations API provides resources for managing an Atlassian organization, with this API, you can get an [audit log of events](https://developer.atlassian.com/cloud/admin/organization/rest/api-group-events/#api-group-events) from organization. This integration collects that data from the Atlassian Organizations API and ingests them into Sumo Logic.
```
Comment (jpipkin1) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/atlassian-source.md:25]:
```suggestion
The Atlassian Organizations API provides resources for managing an Atlassian organization, with this API, you can get an [audit log of events](https://developer.atlassian.com/cloud/admin/organization/rest/api-group-events/#api-group-events) from organizations. This integration collects that data from the Atlassian Organizations API and ingests them into Sumo Logic.
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/atlassian-source.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+## Data collected
+
+| Polling Interval | Data |
+| :--- | :--- |
+| 1 hour |  [Events](https://api.atlassian.com/admin/v1/orgs/%7BorgId%7D/events)|
```
Comment (jpipkin1) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/atlassian-source.md:31]: I got a 401 error when I clicked this link.

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/atlassian-source.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+
+## Setup
+
+### Vendor configuration
+
+Follow the below steps to generate a Bearer access for user configuration:
+
+1. Log in to your [Atlassian admin console](http://admin.atlassian.com/).
+1. After logging in, navigate to **Organizations** tab with your admin console.
```
Comment (jpipkin1) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/atlassian-source.md:40]:
```suggestion
1. After logging in, navigate to the **Organizations** tab.
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/atlassian-source.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+1. Click on **Create API key** in the top right.
+1. Enter a name for the API key.
+1. Configure the expiration date for the key or leave the default value as 1 week.
+1. Click on **Create** and save the API key.
+1. Copy the values of Organization ID and API key.
+1. Select **Done**.
+
+### Source configuration
+
+When you create an Atlassian Source, you add it to a Hosted Collector. Before creating the Source, identify the Hosted Collector you want to use or create a new Hosted Collector. For instructions, see [Configure a Hosted Collector](/docs/send-data/hosted-collectors/configure-hosted-collector).
```
Comment (jpipkin1) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/atlassian-source.md:51]:
```suggestion
When you create an Atlassian Source, you add it to a Hosted Collector. Before creating the Source, identify the Hosted Collector you want to use or create a new Hosted Collector. For instructions, see [Configure a Hosted Collector and Source](/docs/send-data/hosted-collectors/configure-hosted-collector).
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/atlassian-source.md [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+1. **Organizations**. Click the **+Add** button to enter the Organizations you want to associate. Each Organizations needs a API Key value. This is the value that you generated from the [Atlassian platform](#vendor-configuration).
+1. (Optional) The **Polling Interval** is set for 1 hour by default. You can adjust it based on your needs.
+1. (Optional) In **Initial LookBack Time**, enter the first collection start time. Default is 1 day.
+1. (Optional) **Processing Rules for Logs**. Configure any desired filters, such as allowlist, denylist, hash, or mask, as described in [Create a Processing Rule](/docs/send-data/collection/processing-rules/create-processing-rule).
+1. When you are finished configuring the Source, click **Save**.
+
+## JSON Configuration
+
+Sources can be configured using UTF-8 encoded JSON files with the Collector Management API. See [how to use JSON to configure Sources](/docs/send-data/use-json-configure-sources) for details.
```
Comment (jpipkin1) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/atlassian-source.md:70]:
```suggestion
Sources can be configured using UTF-8 encoded JSON files with the Collector Management API. See [Use JSON to Configure Sources](/docs/send-data/use-json-configure-sources) for details. 
```

**Review by jpipkin1 — CHANGES_REQUESTED** [review]
This looks very good, @JV0812. I found only a few minor things.

File: sidebars.ts [hunk 6]
```diff
@@ -258,6 +258,7 @@ module.exports = {
                 'send-data/hosted-collectors/cloud-to-cloud-integration-framework/akamai-siem-api-source',
                 'send-data/hosted-collectors/cloud-to-cloud-integration-framework/armis-api-source',
                 'send-data/hosted-collectors/cloud-to-cloud-integration-framework/asana-source',
+                'send-data/hosted-collectors/cloud-to-cloud-integration-framework/atlassian-source',
```
Comment (jpipkin1) [inline sidebars.ts:261]: If this article is in beta, do we want it to appear in the left nav?

Comment (JV0812) [reply sidebars.ts]: I have hidden this line for now, thanks for identifying this John. There was a confusion on weather to release this source as beta or not, so missed it here.

**Review by jpipkin1 — APPROVED** [review]
Looks great, @JV0812!

**Pull request merged**